shared/locals.tf

```diff
@@ -1,3 +1,5 @@
 locals {
   region = "ap-southeast-1"
+
+  app_name = "devops_ic"
```

Comment:
Are you going to deploy it to the Nimble AWS account? I think it can be more personalized, so it will be clear who is the owner of the resource:
```suggestion
  app_name = "tyson_devops_ic"
```

And also please change owner variable to `Tyson`.

<img width="475" alt="image" src="https://github.com/nvminhtue/tyson_devops_ic/assets/475367/3a5c8c38-57c2-40ba-a21b-0d1ba6118758">

Reply:
I use it on my AWS personal account